src/App.tsx

```diff
@@ -1,5 +1,13 @@
+import { ReactLocation, Router } from '@tanstack/react-location';
+
 const App: React.FC = () => {
-  return <div>The Charity App</div>;
+  const location = new ReactLocation();
+  return (
+    <>
+      The Charity App
+      <Router routes={[{ path: '/' }]} location={location} />
+    </>
+  );
```

Comment:
```suggestion
  return (
     <Router routes={[{ path: '/' }]} location={location}>
        The Charity App
    </Router>
  );
```

why not do this instead?
we get rid of the wrapping `<> ... </>` plus I've seen them doing it this way 
what are your thoughts?

Reply:
like what exactly?
I don't think we have anything as a sibling in the app, maybe just a wrapper